scripts/centos/vmware.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh -eux
```

Comment:
You've broken out a `vmware.sh` but parallels and virtualbox still share a `vmtools.sh` - it's not necessary for this PR but it feels like we should break them each out or put them all on in one like vmtools.sh.